x-pack/test/observability_api_integration/trial/tests/obs_alert_details_context.ts

```diff
@@ -0,0 +1,513 @@
+          expect(logCategory?.docCount).to.be.greaterThan(0);
+          expect(logCategory?.errorCategory).to.be('Error message from service Backend');
+        });
+      });
+
+      describe('when non-existing service.name is specified', async () => {
+        let response: SupertestReturnType<'GET /internal/observability/assistant/alert_details_contextual_insights'>;
+        before(async () => {
+          response = await obsApiClient.adminUser({
+            endpoint: 'GET /internal/observability/assistant/alert_details_contextual_insights',
+            params: {
+              query: {
+                alert_started_at: new Date(end).toISOString(),
+                'service.name': 'non-existing-service',
+              },
+            },
+          });
+        });
+
+        it('returns empty service summary', () => {
+          const serviceSummary = response.body.alertContext.find(
+            ({ key }) => key === 'serviceSummary'
+          );
+          expect(serviceSummary?.data).to.eql({
+            'service.name': 'non-existing-service',
+            'service.environment': [],
+            instances: 1,
+            anomalies: [],
+            alerts: [],
+            deployments: [],
+          });
+        });
+
+        it('does not return log categories', () => {
+          const logCategories = response.body.alertContext.find(
+            ({ key }) => key === 'logCategories'
+          )?.data as LogCategory[];
+          expect(logCategories).to.have.length(1);
+
+          expect(
+            logCategories.map(({ errorCategory }: { errorCategory: string }) => errorCategory)
+          ).to.eql(['Error message from container my-container-c']);
+        });
+      });
+    });
+
+    async function ingestTraces(eventMetadata: {
+      'service.name': string;
+      'container.id'?: string;
+      'host.name'?: string;
+      'kubernetes.pod.name'?: string;
+    }) {
+      const serviceInstance = apm
+        .service({
+          name: eventMetadata['service.name'],
+          environment: 'production',
+          agentName: 'java',
+        })
+        .instance('my-instance');
+
+      const events = range
+        .interval('1m')
+        .rate(1)
+        .generator((timestamp) => {
+          return serviceInstance
+            .transaction({ transactionName: 'tx' })
+            .timestamp(timestamp)
+            .duration(10000)
+            .defaults({ 'service.version': '1.0.0', ...eventMetadata })
+            .outcome('success')
+            .children(
+              serviceInstance
+                .span({
+                  spanName: 'GET apm-*/_search',
+                  spanType: 'db',
+                  spanSubtype: 'elasticsearch',
+                })
+                .duration(1000)
+                .success()
+                .destination('elasticsearch')
+                .timestamp(timestamp)
+            );
+        });
+
+      await apmSynthtraceClient.index(events);
```

Comment:
You don't need `await` here. Just return it and the client should await the response.

```suggestion
      return apmSynthtraceClient.index(events);
```